project.godot

```diff
@@ -145,6 +145,9 @@ PlayerVariables="*res://scenes/PlayerVariables.gd"
 NoteFrequency="*res://autoload/note_frequency.gd"
 InstrumentInput="*res://autoload/instrument_input/instrument_input.tscn"
 Debug="*res://autoload/debug/debug.tscn"
+GLoginDialog="*res://scenes/login_dialog/login_dialog.tscn"
+GRegisterDialog="*res://scenes/register_dialog/register_dialog.tscn"
+GDialogs="*res://scenes/dialogs/dialogs.tscn"
```

Comment:
Not a fan of these being singletons. Better to instantiate windows as you need, optionally using a UI stack to keep track of which windows you've opened. Also, make sure to follow gdscript guidelines, no prefixes. So LoginDialog, RegisterDialog and Dialogs, all without the leading G.